outputs/file/file.go

```diff
@@ -111,6 +116,14 @@ func (f *File) Init(ctx context.Context, cfg map[string]interface{}, opts ...out
 	if f.Cfg.Multiline && f.Cfg.Indent == "" {
 		f.Cfg.Indent = "  "
 	}
+
+	if f.Cfg.ConcurrencyLimit > -1 {
```

Comment:
I think the semaphore should always be initialized, even if `Config.ConcurrencyLimit < 0`, otherwise the `f.sem.Acquire()` on line 138 would panic if `f.sem == nil`
Alternatively, it's possible to wrap the `f.sem.Acquire()`in a ` if f.sem != nil {}`